keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -252,13 +252,11 @@ multiple apiservers.
 
 The new allocation mode requires:
 
-- 2 new API objects ServiceCIDR and IPAddress in networking.k8s.io/v1alpha1, see <https://groups.google.com/g/kubernetes-sig-api-machinery/c/S0KuN_PJYXY/m/573BLOo4EAAJ>. Both will be protected with finalizers.
-- 1 new allocator implementing current `allocator.Interface`, that runs in each apiserver, and uses the new objects to allocate IPs for Services.
-- 1 new controller that participates on the ServiceCIDR deletion, the guarantee that each IPAddresses has
```

Comment:
I think you still want this finalize-step, right?  This prevents CIDR being deleted while IPs exist.

Reply:
yes, thanks for catching it, ServiceCIDR needs finalizers

Reply:
oh,  see below

> The ServiceCIDR will be protected with a finalizer

this is one controller only now, it handles the boostrap and the reconciliation of ServiceCIDRs, the multiClusterCIDRipam controller does the same

> - 1 new controller that handles the bootstrap process and the ServiceCIDR object, it participates on the ServiceCIDR deletion, the guarantee that each IPAddresses has a ServiceCIDR associated.

Reply:
do you want me to reword it?

Reply:
You took out the "participates on the ServiceCIDR deletion ... " text which I thought was valuable.

Reply:
I didn't take it out, I just moved to the last bullet point :)

Reply:
I reworded in the last commit, PTAL